hawkular-actions-plugins/hawkular-actions-email/src/main/java/org/hawkular/actions/email/registration/RegistrationInit.java

```diff
@@ -58,9 +59,13 @@ public void init() {
             ccf = new ConnectionContextFactory(conFactory);
             pcc = ccf.createProducerConnectionContext(new Endpoint(Endpoint.Type.QUEUE, ACTION_PLUGIN_REGISTER));
 
+            /*
+                This is a registration message, it should contain
+             */
+
             ActionPluginMessage apMsg = new ActionPluginMessage();
-            apMsg.setOp("init");
-            apMsg.setActionPlugin("email");
+            apMsg.setOp(EmailPlugin.INIT_PLUGIN);
+            apMsg.setActionPlugin(EmailPlugin.PLUGIN_NAME);
             Set<String> properties = new HashSet<String>();
             properties.add("to");
             properties.add("cc");
```

Comment:
Ops, I thought you were commented the marked lines :-).
Yes, it seems I missed to add the new list of properties supported by the plugin.
Good catch. I will fix it and I will send a new commit with that.
Thx.